content/designer/visual-design.md

```diff
@@ -121,7 +121,7 @@ Students should be able to:
 * determine sufficient color contrast for text, images of text, and controls
 * design text and images of text that have a color contrast of at least 4.5:1 with respect to their background
 * design large-scale text and images of text that have a color contrast of at least 3:1 with respect to their background
-* design components and graphics that have a color contrast of at least 3:1 with respect to their background
+* design user interface components and graphics that have a color contrast of at least 3:1 with respect to their background
```

Comment:
Thanks @iadawn 

We had "user interface components" before. Some of the learning outcomes started with "design user interfaces that ..." My understanding is that phrase made most of the learning outcomes very long and difficult to process. Why do you think "user interface components" is needed here?

Reply:
I think it was the 'components and graphics' got me thinking of 'graphics' as 'images' and that isn't something that would be covered by this. Trying to be a bit more specific in this instance helped it in my head.